src/cloudkit.cs

```diff
@@ -96,18 +273,36 @@ interface CKContainer {
 		[Export ("fetchLongLivedOperationWithID:completionHandler:")]
 		[Async]
 		void FetchLongLivedOperation (string[] operationID, Action<NSDictionary<NSString,NSOperation>, NSError> completionHandler);
-		
+
+		[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12, onlyOn64 : true)]
+		[Export ("fetchShareParticipantWithEmailAddress:completionHandler:")]
+		[Async]
+		void FetchShareParticipantWithEmailAddress (string emailAddress, Action<CKShareParticipant, NSError> completionHandler);
```

Comment:
That is exactly the reason why I left it like that, I can't think of a nice way to deal with this without overclomplicating things.